mono/mini/exceptions-amd64.c

```diff
@@ -867,11 +867,15 @@ prepare_for_guard_pages (MonoContext *mctx)
 }
 
 static void
-altstack_handle_and_restore (MonoContext *ctx, MonoObject *obj, gboolean stack_ovf)
+altstack_handle_and_restore (MonoContext *ctx, MonoObject *obj, guint32 flags)
 {
 	MonoContext mctx;
-	MonoJitInfo *ji = mini_jit_info_table_find (mono_domain_get (), MONO_CONTEXT_GET_IP (ctx), NULL);
+	MonoJitInfo *ji = NULL;
+	gboolean stack_ovf = (flags & 1) != 0;
+	gboolean native = (flags & 2) != 0;
 
+	if (!native)
+		ji = mini_jit_info_table_find (mono_domain_get (), MONO_CONTEXT_GET_IP (ctx), NULL);
```

Comment:
shouldn't `mini_jit_info_table_find ()` return `NULL` for instruction pointers that are not jitted? thus, implying that it's C code? Assuming that was meant with `native`.